### PR TITLE
3.2.4 Release

### PR DESCRIPTION
**Pull request by jrtibbetts**
[8441: iOS SDK 3.2.4 release](https://app.shortcut.com/radarlabs/story/8441/ios-sdk-3-2-4-release)

Bumped the SDK release number to 3.2.4, then regenerated the XCFramework and docs.

This release contains the following changes:

* [Add guards against stopping CLVisits and SLCs from initializing Radar (SDK P1)](https://app.shortcut.com/radarlabs/story/8335/add-guards-against-stopping-clvisits-and-slcs-from-initializing-radar-sdk-p1) (PR #140)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -45,6 +45,6 @@ build-example-pretty:
 docs:
 	jazzy
 
-dist: clean-pretty format test-pretty build-pretty docs lint
+dist: clean-pretty test-pretty build-pretty lint jazzy
```
Comment (tjulien) [inline Makefile:48]: small thing - are we intending to not run `format` going forward?

Comment (jrtibbetts) [reply Makefile:48]: Good question. When I run it, it does weird things to the indentation; when Nick runs it, it's fine. He's using an older version, I think. Maybe I can roll mine back using Homebrew. Whatever the case, it directly conflicts with Xcode's own formatting rules (ctrl+I). I can run it and then manually undo some of the weird changes.

Comment (tjulien) [reply Makefile:48]: Got it, I think it's fine to remove it for now given that the latest version of format isn't doing what we want.  I'll add something to the SDK roadmap to figure out what we want to do here going forward.

Comment (jrtibbetts) [reply Makefile:48]: For this release, I recommend just skipping it.

**Review by tjulien — APPROVED** [review]

**Pull request merged**